tools/run-fbp-tests

```diff
@@ -68,26 +68,38 @@ def collect_tests(tests):
             result.append(t)
     return result
 
-# TODO: Consider putting the expected output as comments inside the
-# input file. This would also allow specifying other parameters there
-# like the expected error code, avoiding the clumsyness of two
-# different expected files.
 def get_expected(t):
     code = 0
     out = None
-    no_ext = os.path.splitext(t)[0]
-    success_file = no_ext + "-expected.txt"
-    fail_file = no_ext + "-expected-fail.txt"
+    skip = None
 
-    if os.path.isfile(success_file):
-        with open(success_file) as f:
-            out = f.read()
-    elif os.path.isfile(fail_file):
-        with open(fail_file) as f:
-            out = f.read()
-        code = 1
+    f = open(t)
+    for line in f:
+        if not line.startswith("## "):
```

Comment:
If the line doesn't start with `## ` (with space) we won't treat it as a command.